packages/apps/auth0-actions/src/verification-email.html

```diff
@@ -0,0 +1,64 @@
+    </style>
+  </head>
+  <body>
+    <center>
+      <table
+        style='width: 600px;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;mso-table-lspace: 0pt;mso-table-rspace: 0pt;margin: 0;padding: 0;font-family: "ProximaNova", sans-serif;border-collapse: collapse !important;height: 100% !important;'
+        align="center"
+        border="0"
+        cellpadding="0"
+        cellspacing="0"
+        height="100%"
+        width="100%"
+        id="bodyTable"
+      >
+        <tr>
+          <td
+            align="center"
+            valign="top"
+            id="bodyCell"
+            style='-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;mso-table-lspace: 0pt;mso-table-rspace: 0pt;margin: 0;padding: 20px;font-family: "ProximaNova", sans-serif;height: 100% !important;'
+          >
+            <div class="main">
+              <p
+                style="text-align: center;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; margin-bottom: 30px;"
+              >
+                <img
+                  src="https://identity-public-assets-stage.s3.eu-west-1.amazonaws.com/images/wellcomecollections-150x50.png"
+                  width="200"
+                  alt="Wellcome Collection Logo"
+                  style="-ms-interpolation-mode: bicubic;border: 0;height: auto;line-height: 100%;outline: none;text-decoration: none;"
+                />
+              </p>
+
+              <p>Thank you for signing up. Please verify your email address by clicking the following link:</p>
+
+              <p><a href="{{ url }}">Confirm my account</a></p>
+
+              <p>
+                If you are having any issues with your account, please don’t hesitate to contact us by replying to
```

Comment:
There's actually support details configured on the tenant level for this purpose, available as {{ support_email }} and {{ support_url }} - I would suggest we reference that here instead as there's no guarantee that the address from which this email is sent will be able to receive replies.